includes/class-nrh.php

```diff
@@ -73,9 +87,15 @@ public static function update_settings( $data ) {
 		foreach ( $data as $key => $value ) {
 			if ( in_array( $key, self::$allowed_keys, true ) ) {
 				$settings[ $key ] = $value;
+			} elseif ( 'newspack_popups_donor_landing_page' === $key && class_exists( '\Newspack_Popups_Settings' ) ) {
+				// Update the donor landing page in Campaigns settings.
+				\update_option( $key, $value );
```

Comment:
I know we are duplicating the UI here, but I wouldn't also duplicate the logic that stores the option in the database. It's better to call a method in the popups plugin itself and let it handle the read and update for this option. wdyt?

Reply:
61efac8 and 473daeb should implement this (and [dbdca47](https://github.com/Automattic/newspack-popups/pull/1124/commits/dbdca477a1e37029607658a4289869370113061e) in the other repo). Because of the value validation in the Campaigns plugin, I had to reconfigure things a bit on the main plugin's wizard API callback.